migrations/20220201085056_pulling_data/migration.sql

```diff
@@ -1 +0,0 @@
--- This is an empty migration.
```

Comment:
This change is not related to the PR, please undo it

Reply:
Done.